src/components/Modal/SendToken.vue

```diff
@@ -23,7 +24,20 @@ const showPicker = ref(false);
 const searchValue = ref('');
 const { loading, loaded, assets, assetsMap, loadBalances } = useBalances();
 
-const currentToken = computed(() => assetsMap.value?.get(form.token));
+const currentToken = computed(() => {
+  const token = assetsMap.value?.get(form.token);
+
+  if (!token) {
+    return {
+      contract_decimals: 18,
+      contract_name: 'Ether',
+      contract_ticker_symbol: 'ETH',
+      contract_address: ETH_CONTRACT
+    };
+  }
```

Comment:
Fallback to static token as we only load list of assets when user actually opens the picker.

Reply:
@bonustrack just noticed that because it's missing quote, to USD value input is not going to work unless you open token picker first.
Should we fetch all balances when modal loads to avoid this? I'm not sure how can we do that otherwise (unless we somehow only load ETH rate here to make inputs work as expected).